database/metadata/databases/default/tables/tables.yaml

```diff
@@ -14,6 +14,7 @@
 - "!include lookups_collsn.yaml"
 - "!include lookups_contrib_factr.yaml"
 - "!include lookups_drvr_ethncty.yaml"
+- "!include lookups_drvr_lic_type.yaml"
```

Comment:
This was a surprise -- I'm not sure why were not tracking this before. 🤷 Open to any ideas or explanations

Reply:
it was a surprise to me too!

this table was added via https://github.com/cityofaustin/vision-zero/pull/1546, and it was an oversight to not track Hasura select permissions at that time. we were getting by without tracking because this table is not used by the VZE.

i'll update our DB docs with a reminder to track new lookup tables so that this script doesn't get confused.